tests/distributions/test_grouped_normal_normal.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import math
+
+import torch
+
+from pyro.distributions import GroupedNormalNormal, Normal
+from tests.common import assert_close
+
+
+def test_grouped_normal_normal(num_groups=3, num_samples=10 * 6):
```

Comment:
did you mean `10 ** 6`?  would that allow tighter `atol` below?

Reply:
good catch 👍